bin/waagent2.0

```diff
@@ -80,7 +80,7 @@ if not hasattr(subprocess,'check_output'):
 
     subprocess.check_output=check_output
     subprocess.CalledProcessError=CalledProcessError
-    
```

Comment:
We have stopped supporting the waagent2.0 in the Guest Agent, its still there only for the older extensions. Each extension that had a dependency on this code has their own copy of the waagent in their repo. Why are these changes needed?